src/libraries/System.IO.Pipelines/src/System/IO/Pipelines/PipeReader.cs

```diff
@@ -15,7 +15,15 @@ public abstract partial class PipeReader
         /// <summary>Attempts to synchronously read data from the <see cref="System.IO.Pipelines.PipeReader" />.</summary>
         /// <param name="result">When this method returns <see langword="true" />, this value is set to a <see cref="System.IO.Pipelines.ReadResult" /> instance that represents the result of the read call; otherwise, this value is set to <see langword="default" />.</param>
         /// <returns><see langword="true" /> if data was available, or if the call was canceled or the writer was completed; otherwise, <see langword="false" />.</returns>
-        /// <remarks>If the pipe returns <see langword="false" />, there is no need to call <see cref="System.IO.Pipelines.PipeReader.AdvanceTo(System.SequencePosition,System.SequencePosition)" />.</remarks>
+        /// <remarks>
+        /// If the pipe returns <see langword="false" />, there is no need to call <see cref="System.IO.Pipelines.PipeReader.AdvanceTo(System.SequencePosition,System.SequencePosition)" />.
+        /// <para>
+        /// Note: the <see cref="System.IO.Pipelines.PipeReader" /> implementation returned by <see cref="System.IO.Pipelines.PipeReader.Create(System.IO.Stream, System.IO.Pipelines.StreamPipeReaderOptions?)"/>
```

Comment:
@dotnet/docs do we need to quote this and add a `[!NOTE]` for this to show up correctly?

I think so looking at [this](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Threading/ThreadPool.xml#L85-L91)

Reply:
I don't know that you can use `[!NOTE]` because that's markdown. /// comments are HTML. I believe the only way to use markdown and the ms.docs syntax extensions (which `[!NOTE]` is) is to migrate this content to the external XML API files.

Tagging @BillWagner @gewarren for comment.

Reply:
You'll need to wrap it in `<format>` tags like this: https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/System.IO.Pipelines/src/System/IO/Pipelines/PipeWriter.cs#L46.